### PR TITLE
Slate updates

### PR DESCRIPTION
**Pull request by quininez**
## Description
* remove dead code related to tags inside the slate editor
* update slate and related packages to latest versions
* transform slate editor into an uncontrolled component
* expose editor children (the actual content) through useImperativeRef

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: extensions/apps/antenna/src/components/app-routes/index.tsx [hunk 1]
```diff
@@ -39,6 +39,9 @@ import {
 } from './data-loaders';
 import { mapReflectEntryData } from '@akashaorg/ui-awf-hooks';
 import { NotFoundComponent } from './not-found-component';
+import { EditorActions } from '@akashaorg/design-system-components/lib/components/Editor';
+
+const editorActionsRef = createRef<EditorActions>();
```
Comment (didd) [inline extensions/apps/antenna/src/components/app-routes/index.tsx:44]: I don't think we need this ref here as it's not used.

File: extensions/apps/antenna/src/components/reflect-editor/index.tsx [hunk 2]
```diff
@@ -128,7 +128,7 @@ const ReflectEditor: React.FC<ReflectEditorProps> = props => {
     const response = await publishReflection({
       variables: {
         i: {
-          content,
+          content: content,
```
Comment (didd) [inline extensions/apps/antenna/src/components/reflect-editor/index.tsx]:
```suggestion
          content,
```
It's minor thing :)

File: extensions/apps/antenna/src/components/app-routes/index.tsx (hunk 1 above)
Comment (quininez) [reply extensions/apps/antenna/src/components/app-routes/index.tsx:44]: it is used, it's passed to the reflect editor

Comment (didd) [reply extensions/apps/antenna/src/components/app-routes/index.tsx:44]: I meant the actions associated with the ref aren't used  and I wondered if the ref is needed here :)

**Review by didd — DISMISSED** [review]

File: libs/design-system-components/src/components/Editor/formatting-buttons.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ export const MarkButton: React.FC<ToolbarButtonProps> = ({ format, icon, style }
   const active = CustomEditor.isMarkActive(editor, format);
   return (
     <button
-      onClick={event => {
+      onMouseDown={event => {
```
Comment (kenshyx) [inline libs/design-system-components/src/components/Editor/formatting-buttons.tsx]: this event does not trigger on mobile devices

Comment (didd) [reply libs/design-system-components/src/components/Editor/formatting-buttons.tsx:47]: Thats right

Comment (quininez) [reply libs/design-system-components/src/components/Editor/formatting-buttons.tsx]: good catch, I changed it to test something and forgot to revert

File: extensions/apps/antenna/src/components/app-routes/index.tsx (hunk 1 above)
Comment (quininez) [reply extensions/apps/antenna/src/components/app-routes/index.tsx:44]: they are used in the reflect editor, to check if editor is empty or not through the children that are exposed on that ref

**Review by didd — APPROVED** [review]

**Review by kenshyx — APPROVED** [review]

**Pull request merged**